modules/learn/pages/clusters-and-availability/xdcr-conflict-logging-feature.adoc

```diff
@@ -0,0 +1,585 @@
+   "clusterUUID": "1bc375a26b7f822ebe91288c58a6c614",
+   "collection": "_default",
+   "expiry": 0,
+   "flags": 0,
+   "id": "src_1749454000005461807_e2f416ed317beb4f16a3587656b3d3450e1bec893cabebd3b9bb857d63824da4",
+   "isDeleted": false,
+   "nodeId": "10.0.32.102",
+   "revSeqno": 2,
+   "scope": "_default",
+   "xattrs": {
+     "_mou": "",
+     "_sync": "",
+     "_vv": ""
+   }
+ },
+ "tgtDoc": {
+   "bucketUUID": "6cd3a35ba2d337d84d4fcd30c3a46fb9",
+   "cas": 1749453999840231400,
+   "clusterUUID": "d976db26a5ce512a243db65a3026073f",
+   "collection": "_default",
+   "expiry": 0,
+   "flags": 0,
+   "id": "tgt_1749454000005461807_e2f416ed317beb4f16a3587656b3d3450e1bec893cabebd3b9bb857d63824da4",
+   "isDeleted": false,
+   "nodeId": "10.0.25.110",
+   "revSeqno": 2,
+   "scope": "_default",
+   "xattrs": {
+     "_mou": "",
+     "_sync": "",
+     "_vv": "{\"cvCas\":\"0x00003dca244f4718\",\"src\":\"ntR6bcR9VHcjcPpQ5osEiw\",\"ver\":\"0x00003dca244f4718\"}"
+   }
+ },
+ "timestamp": "2025-06-09T07:26:40.005Z"
+}
+----
+
+Some of the key-value pairs in CRD are as follows:
+
+|===
+| Key | Value
+
+| `timestamp`
+| Date and time of the event in the UTC ISO 8601 format:
+YYYY-MM-DDThh:mm:ss:SSSZ
+
+The timestamp is the wall clock of the node that noted the conflict.
+
+| `id`/`srcDoc.id`/`tgtDoc.id`
+| CRD document ID prefixed with `crd_`.
+
+A copy of conflict source document ID prefixed with `src_`.
+
+A copy of conflict target document ID prefixed with `tgt_`.
+
+| `docId`
+| document id of the actual document in conflict.
+
+| `replId`
+| Replication ID.
+
+| `srcDoc` and `tgtDoc`
+These JSON Objects have the IDs of the source and target document copies in the conflict collection.
+The document IDs are indicated by `srcDoc.id` and `tgtDoc.id`.
+
+| `nodeId`
+| Node identification, which can be the Hostname or IP address and a port.
+
+| `bucketUUID`
+| Source/target bucket unique ID.
+
+| `cas`
+| Source/target document CAS.
+
+| `clusterUUID`
+| Source/target cluster unique ID.
+
+| `isDeleted`
+| True or false for - is the source/target document deleted?
+
+| `scope` and `collection`
+| Name of the scope and collection where the conflict source/target documents exist.
+
+| `revSeqno`
+| Source/target document’s revision sequence number.
+
+| `expiry`
+| Source/target document’s expiry time.
+
+| `flags`
+| Source/target document’s flag.
+
+| `xattrs`
+| System xattributes.
+
+| `_vv`
+| Document version vector.
+HLV metadata.
+|===
+
+[#conflict-logger]
+=== Conflict Logger
+
+The XDCR Conflict Logger logs conflicts continuously and is designed to minimize the impact on replication performance and system resources.
+
+Conflict logging is performed on a best-effort basis.
+If system resources are limited or errors occur while writing to the conflict log collection, XDCR prioritizes replication and High Availability.
+In these cases, temporarily the conflict logging may be skipped or the conflict logs are dropped.
```

Comment:
This sentence sounds a bit odd.  Maybe the sentence below is more clear.

In these cases, conflict logging may be stopped temporarily (this means that some conflicts will not be logged).